src/main/java/exceptions/NoActivityException.java

```diff
@@ -0,0 +1,10 @@
+package exceptions;
+
+public class NoActivityException extends Exception {
```

Comment:
make exceptions different from each other, so there is some difference except for name

Reply:
fixed.